### PR TITLE
[#3850] improvement(docker-image): Change Kerberos Hive image to add more principals and key tables

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

Add more proxy users in the file `core-site.xml` in the Kerberos Hive docker file

### Why are the changes needed?

As we are going to support schema or fileset level user authentication, we need more principals and key tables, so we have to change the docker image file.

Fix: #3850 

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

**Comment by yuqi1129** [comment]
@qqqttt123 
Could you please help me review this?

File: docs/docker-image-details.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ You can use these kinds of Docker images to facilitate integration testing of al
 You can use this kind of image to test the catalog of Apache Hive with kerberos enable
 
 Changelog
+- gravitino-ci-kerberos-hive:0.1.2
```
Comment (jerqi) [inline docs/docker-image-details.md:100]: 0.1.3?

Comment (yuqi1129) [reply docs/docker-image-details.md]: fix

**Review by jerqi — APPROVED** [review]
LGTM.

**Review by qqqttt123 — APPROVED** [review]

**Pull request merged**